src/document_to_podcast/inference/text_to_speech.py

```diff
@@ -1,27 +1,72 @@
+from typing import Union
+
 import numpy as np
-from transformers import PreTrainedModel, PreTrainedTokenizerBase
+from outetts.version.v1.interface import InterfaceGGUF
+from transformers import PreTrainedTokenizerBase, PreTrainedModel
 
 
-def text_to_speech(
+def _speech_generation_oute(
```

Comment:
No strong opinion but why not use the same name as the main function:

```suggestion
def _text_to_speech_oute(
```